vuu-ui/packages/vuu-shell/src/layout-management/useLayoutManager.tsx

```diff
@@ -1,31 +1,36 @@
 import React, {
-  useState,
   useCallback,
   useContext,
   useEffect,
   useRef,
+  useState,
 } from "react";
 import {
   defaultLayout,
   LayoutJSON,
   LayoutPersistenceManager,
   LocalLayoutPersistenceManager,
+  RemoteLayoutPersistenceManager,
   resolveJSONPath,
 } from "@finos/vuu-layout";
-import { LayoutMetadata } from "./layoutTypes";
+import { LayoutMetadata, LayoutMetadataDto } from "./layoutTypes";
+
+const local = process.env.LOCAL ?? true;
```

Comment:
need to make add entry for this process-env in esbuild.mjs in top level scripts

Reply:
We've raised a [ticket](https://github.com/ScottLogic/finos-vuu/issues/113) to deal with environment variables. It would be good to go over this with you some time next week.

Reply:
Done! [9d060cf](https://github.com/finos/vuu/pull/970/commits/9d060cfa76f0abb7f2aefd5e1c3ff93c1fbbf9d8)